<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuModels">
    <property name="title">
     <string>Models</string>
    </property>
    <addaction name="action_Pla_Base"/>
    <addaction name="action_Bola_Blanca"/>
    <addaction name="action_15_Boles"/>
    <addaction name="action_Obre_Fitxer"/>
    <addaction name="action_Sala_de_Billar"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuPlay">
    <property name="title">
     <string>Controls</string>
    </property>
    <addaction name="action_Play"/>
   </widget>
   <addaction name="menuModels"/>
   <addaction name="menuPlay"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Play">
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="action_Pla_Base">
   <property name="text">
    <string>Afegeix Pla Base</string>
   </property>
  </action>
  <action name="action_Bola_Blanca">
   <property name="text">
    <string>Afegeix Bola Blanca</string>
   </property>
  </action>
  <action name="action_15_Boles">
   <property name="text">
    <string>Afegeix 15 Boles</string>
   </property>
  </action>
  <action name="action_Obre_Fitxer">
   <property name="text">
    <string>Obre Fitxer</string>
   </property>
   <property name="toolTip">
    <string>Obre Fitxer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_Sala_de_Billar">
   <property name="text">
    <string>Afegeix Sala de Billar</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
